### PR TITLE
CSAT for host/deploy to IIS

### LINKED ISSUE #17718: Improve CSAT OKR: Host ASP.NET Core on Windows with IIS
[Internal bug](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1702032)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec484002-18c1-6a0d-00de-820d56f038d3
* Version Independent ID: 7903ce53-2cbb-f083-7479-c8abc30211a0
* Content: [Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1)
* Content Source: [aspnetcore/host-and-deploy/iis/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/iis/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #17718, be sure to update internal issue when this merges.

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1&branch=pr-en-us-18165)

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -671,7 +671,11 @@ Use a 64-bit (x64) .NET Core SDK to publish a 64-bit app. A 64-bit runtime must
 
 ### In-process hosting model
 
-Using in-process hosting, an ASP.NET Core app runs in the same process as its IIS worker process. In-process hosting provides improved performance over out-of-process hosting because requests aren't proxied over the loopback adapter, a network interface that returns outgoing network traffic back to the same machine. IIS handles process management with the [Windows Process Activation Service (WAS)](/iis/manage/provisioning-and-managing-iis/features-of-the-windows-process-activation-service-was).
+Using in-process hosting, an ASP.NET Core app runs in the same process as its IIS worker process. In-process hosting provides improved performance over out-of-process hosting because:
+
+* Requests aren't proxied over the loopback adapter. A loopback adapter is a network interface that returns outgoing network traffic back to the same machine.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/index.md:676]: @guardrex  please advise

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md:690]: What's the question?

File: aspnetcore/toc.yml [hunk 2]
```diff
@@ -804,7 +804,7 @@
     - name: IIS
       displayName: deploy, publish
       items:
-        - name: Overview
+        - name: Host and deploy on IIS
```
Comment (guardrex) [inline aspnetcore/toc.yml:807]: Is this a one-off, or is there new guidance for naming the first topic of a node?

Comment (Rick-Anderson) [reply aspnetcore/toc.yml]: Arg, reverted that. I read it wrong.

**Pull request merged**